modules/tinymce/CHANGELOG.md

```diff
@@ -85,6 +85,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Drag and dropping the last noneditable element out of its parent block would not properly pad the parent block element. #TINY-9606
 - Applying heading formats from `text_patterns` produced an invisible space before a word. #TINY-9603
 - Opening color swatches would cause tab to crash when `color_cols` or other column option was set to 0. #TINY-9649
+- Table toolbar was visible even if the table was within a noneditable host element. #TINY-9664
```

Comment:
This will be moved to the right spot after the 6.4 release. I stuck it here until that happens.

Reply:
This is now ready to go